Gems/Terrain/Code/Source/TerrainRenderer/Vector2i.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <TerrainRenderer/Vector2i.h>
```

Comment:
Suggestion:  Consider moving Vector2i.* to Source/TerrainSystem or Source/Utils

Reply:
Will do this and Aabb2i work in separate PR.